custom/aaa/static/aaa/js/models/pregnant_women.js

```diff
@@ -188,6 +187,14 @@ hqDefine("aaa/js/models/pregnant_women", [
             return reachUtils.BLOODGROUPS[self.bloodGroup()];
         });
 
+        self.abortionWeeks = ko.computed(function () {
+            if (self.abortionDays() !== void(0)) {
```

Comment:
```suggestion
            if (self.abortionDays() !== undefined) {
```

Reply:
@emord could you explain why? ```undefined``` is a global variable which we can override, so using ```void(0)``` is safer

Reply:
> In modern browsers (JavaScript 1.8.5 / Firefox 4+), undefined is a non-configurable, non-writable property per the ECMAScript 5 specification. Even when this is not the case, avoid overriding it.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined#Description

Reply:
@emord Thanks, in the past one of dimagi dev suggest that we should use ```void(0)``` in this comparison so that's why I asking,